include/drivers/flash.h

```diff
@@ -36,12 +36,20 @@ struct flash_pages_layout {
 };
 #endif /* CONFIG_FLASH_PAGE_LAYOUT */
 
+/**
+ * Flash memory parameters
+ */
+struct flash_parameters {
+	uint8_t erase_value; /* Pattern representing of erased flash */
```

Comment:
I guess it should be either
`Pattern representation of erased flash`
or
`Pattern representing erased flash` 
?

Reply:
Or "byte value read back from erased area"